src/Cms/App.php

```diff
@@ -365,16 +365,17 @@ public function clone(array $props = [], bool $setInstance = true)
 	 * automatically injected
 	 *
 	 * @param string $name
+	 * @param string $options
 	 * @return \Kirby\Cms\Collection|null
 	 */
-	public function collection(string $name)
+	public function collection(string $name, array $options = [])
 	{
-		return $this->collections()->get($name, [
+		return $this->collections()->get($name, array_merge($options, [
 			'kirby' => $this,
 			'site'  => $this->site(),
 			'pages' => $this->site()->children(),
 			'users' => $this->users()
-		]);
+		]));
```

Comment:
Is it intended that `$options` cannot overwrite these defaults?

Reply:
It was the other way around originally, but I suggested to implement it like this: https://github.com/getkirby/kirby/pull/5081#discussion_r1119828638

Do you see a use case to override the core objects?

Reply:
I just found it a bit odd as we usually allow defaults to be overwritten by specifically provided options. But have no strong feelings for this particular use case.